samples/drivers/flash_shell/test_shell.yml

```diff
@@ -0,0 +1,3 @@
+test_shell_harness:
+- command: "flash page_info 0"
+  expected: "Page for address 0x0"
```

Comment:
`Page for address 0x0:`

Reply:
![image](https://github.com/user-attachments/assets/0a3d9b70-3179-40d8-80db-5d68b4808647)
does the ':' in the end matter, or the "expected" will be matched without?